Guidelines.md

```diff
@@ -1338,13 +1338,10 @@ In this scenario the _databases_ segment is processing the PUT operation.
 
 ```http
 HTTP/1.1 202 Accepted
-Operation-Location: https://api.contoso.com/v1.0/operations/123
+Location: https://api.contoso.com/v1.0/operations/123
 ```
 
-For services that need to return a 201 Created here, use the hybrid flow described below.
-
-The 202 Accepted should return no body.
```

Comment:
These requirements seem important. Not clear why they are removed.

Reply:
This PR proposes that a 202 also be used for the hybrid flow and during that scenario a partially created response is returned.  Therefore saying that a body should not be returned with a 202 would be contradictory.